src/components/Calculator.js

```diff
@@ -1,30 +1,48 @@
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```

Comment:
- [ ] it would be better to remove the commented code in order to keep the file clean and pro

Reply:
resolved! ✔️